src/components/v2v3/V2V3Project/ProjectDashboard/hooks/useTokensPanel.ts

```diff
@@ -16,7 +17,18 @@ export const useTokensPanel = () => {
     tokenSymbol: tokenSymbolRaw,
     totalTokenSupply,
     tokenAddress,
+    fundingCycleMetadata,
   } = useProjectContext()
+
+  const { data: undistributedReservedTokens } = useProjectReservedTokens({
+    projectId,
+    reservedRate: fundingCycleMetadata?.reservedRate,
+  })
+
+  const totalTokenSupplyIncludingReserved = undistributedReservedTokens
```

Comment:
**suggestion (non-blocking)** contracts call this totalOutstandingTokens

https://github.com/jbx-protocol/juice-contracts-v3/blob/5ae86dba2f27dee2dfe2235d20c3a070b7413ff6/contracts/JBController.sol#L167-L187